secrets/secrets.go

```diff
@@ -40,9 +50,9 @@ type secretStore struct {
 // A compile-time constraint to ensure secretStore implements mint.SecretStore.
 var _ mint.SecretStore = (*secretStore)(nil)
 
-// newSecretStore instantiates a new LSAT secrets store backed by an etcd
+// NewSecretStore instantiates a new LSAT secrets store backed by an etcd
 // cluster.
-func newSecretStore(client *clientv3.Client) *secretStore {
+func NewSecretStore(client *clientv3.Client) *secretStore {
```

Comment:
to avoid stuttering (`secretes.NewSecretStore(...)`), how about just `NewStore`?